contracts/margined_insurance_fund/src/state.rs

```diff
@@ -1,121 +1,91 @@
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-use cosmwasm_std::{Addr, Deps, DepsMut, Order, StdError, StdResult, Storage};
+use cosmwasm_std::{Addr, Deps, DepsMut, StdError, StdResult, Storage};
 use cosmwasm_storage::{singleton, singleton_read};
-use cw_storage_plus::Map;
+use cw_storage_plus::Item;
 
 pub static KEY_CONFIG: &[u8] = b"config";
-pub const VAMM_LIST: Map<&Addr, bool> = Map::new("vamm-list");
-pub const VAMM_LIMIT: usize = 10usize;
+pub const VAMM_LIST: Item<Vec<Addr>> = Item::new("vamm-list");
```

Comment:
Previously VAMM_LIST was Item<vammlist> where vammlist was a wrapper around Vec<Addr> - might want to consider something similar